apps/vr-tests-react-components/package.json

```diff
@@ -38,6 +38,7 @@
     "@fluentui/react-menu": "^9.6.2",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26377&commitId=036965c921c97286094710112f930f078e59a680&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1773 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Portal.Apply Class Names.chromium.png|0| Added|
|Portal.Steps.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>